packages/fortinet_fortigate/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.25.1"
+  changes:
+    - description: Make sure we always copy the original_message
```

Maybe something like this.

```suggestion
    - description: Ensure event.original matches the value of the message field.
```

It's good to call out the fields explicitly, if possible.